mkdocs.yml

```diff
@@ -23,7 +23,7 @@ nav:
     - Framework: intro/framework.md
     - Glossary: intro/glossary.md
   - USER GUIDE:
-    - Get Agent: user-guide/get-agent.md
+    - VPP Agent Setup: user-guide/get-vpp-agent.md
```

Comment:
This will require some redirect, so anything that links to `get-agent` will be redirected to `get-vpp-agent`.

Reply:
Right - I'll fix now and resubmit. I'll check Ligato.io and contivpp.io site too.

Reply:
Checked and nothing appeared. Looks okay on my laptop. Can we merge for now? I have a ton to do and would like to begin on the next one.